package.json

```diff
@@ -21,10 +21,22 @@
     "babel-cli": "^6.18.0",
     "babel-plugin-transform-runtime": "^6.15.0",
     "babel-preset-es2015": "^6.18.0",
-    "babel-preset-stage-0": "^6.16.0"
+    "babel-preset-stage-0": "^6.16.0",
+    "stylus": "^0.54.5",
+    "stylus-loader": "^2.4.0",
+    "webpack-dev-server": "^1.16.2"
   },
   "dependencies": {
+    "babel-loader": "^6.2.8",
     "babel-polyfill": "^6.16.0",
-    "babel-runtime": "^6.18.0"
+    "babel-runtime": "^6.18.0",
+    "css-loader": "^0.26.0",
```

Comment:
dependencies which not needed for prod should be moved to save dependencies part.